sources/3.0.0/sections/10-data_product_format.adoc

```diff
@@ -654,9 +666,70 @@ For example, "latitude,longitude". Reverse scan direction along an axis is indic
 
 |===
 
-==== Feature Instance group -- BathymetryCoverage.01
+[[note13]]
+[NOTE]
+====
+The number depends on the number of different vertical datum areas in the Feature Container Group in the Feature Container Group.
```

Comment:
Is this correct?

Reply:
I think that is correct. Let's go through it with an example.

1 Coverage with uniform VDatum (rootGroup) -> 1 VDatum (root) -> numInstances=1
2 Coverage different VDatums -> 2 VDatums (root+BC.02) -> numInstances=2
3 Coverage different VDatums -> 3 VDatums (root+BC.02+BC.03) -> numInstances=3

All areas with a uniform VDatum are combined in one coverage.

![MVDatum](https://github.com/iho-ohi/S-102-Product-Specification/assets/85698889/9e9d98c3-50ec-4899-bc69-3425a4f859a4)

Reply:
Oh wait.

Theoretically, my previous comment is correct.
Unfortunately, the domainExtent.polygon doesn't do that.
The domainExtent.polygon only supports a single "linear ring".
So it is impossible to map multi-polygons, even inner rings are not possible.
Meaning:
Each VDatum area will probably have to become a separate coverage, even if the VDatum is identical.

@PalmBSH will look at this again on Monday.
Good catch, thank you. That's why reviews are important. ;)

![MVDatum_Corrected](https://github.com/iho-ohi/S-102-Product-Specification/assets/85698889/16fca160-7e19-4700-b08d-19310704499a)

Reply:
done in dcd080e1b3cc7df509b1836ce3789d8a5d77ee53